packages/schematics/angular/secondary-entrypoint/schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "SchematicsLibrary",
+  "title": "Secondary Entrypoint Schema",
+  "type": "object",
+  "description": "Creates a secondary entrypoint in a library project in a project.",
+  "additionalProperties": false,
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "The name of the library.",
+      "pattern": "^(?:@[a-zA-Z0-9-*~][a-zA-Z0-9-*._~]*/)?[a-zA-Z0-9-~][a-zA-Z0-9-._~]*$",
+      "$default": {
+        "$source": "argv",
+        "index": 0
+      },
+      "x-prompt": "What name would you like to use for the secondary entrypoint?"
+    },
+    "project": {
+      "type": "string",
+      "description": "The name of the project.",
+      "$default": {
+        "$source": "projectName"
+      }
+    },
+    "entryFile": {
```

Comment:
We probably can opt this option as I doubt it's actually used.

Reply:
You mean the path of the entryfile? Or do you want me to change this so that it is required and consumers should enter a path?

Reply:
I mean the option should be removed.